src/udiskslinuxloop.c

```diff
@@ -122,50 +124,76 @@ udisks_linux_loop_update (UDisksLinuxLoop        *loop,
   UDisksState *state;
   UDisksLinuxDevice *device;
   uid_t setup_by_uid;
+  gchar *backing_file;
+  GError *error;
+  error = NULL;
 
   daemon = udisks_linux_block_object_get_daemon (UDISKS_LINUX_BLOCK_OBJECT (object));
   state = udisks_daemon_get_state (daemon);
-
   device = udisks_linux_block_object_get_device (object);
+
   if (g_str_has_prefix (g_udev_device_get_name (device->udev_device), "loop"))
     {
+
       gchar *filename;
-      gchar *backing_file;
-      GError *error;
+      gchar *backing_file_org;
       filename = g_strconcat (g_udev_device_get_sysfs_path (device->udev_device), "/loop/backing_file", NULL);
-      error = NULL;
-      if (!g_file_get_contents (filename,
-                                &backing_file,
+
+      g_file_get_contents (filename,
+                                &backing_file_org,
                                 NULL,
-                                &error))
+                                &error);
+      udisks_warning ("MYDEBUG _update: backing file org is '%s'", backing_file_org);
```

Comment:
Please remove the debugging stuff and the original code for getting backing file.